### ISSUE TITLE
Exporting and Turning in for Unity

### ISSUE THREAD

**Issue by steven-king** [steven-king/mj588#20]
If you need a refresher, this video is very complete, pick up about 5 minutes in.
https://www.youtube.com/watch?v=7nxKAtxGSn8

Put all your files in a folder with Lastname_FirstName. 
Place your name folder on the Oculus Machine in the MJ588_Spring folder that will be on the desktop.

**Comment by cpa18** [comment]
I am almost finished with my project, but I'm having trouble exporting. First of all, when I click File>Build Settings, the only option I have under "PC, Mac & Linux Standalone" is Mac OS X. How do I do this for PC? Is anyone else having this problem?

**Comment by brianbatista** [comment]
Download the plugin for PC build.

**Comment by rbrandonc** [comment]
Totally forgot, is this due tonight or tomorrow night by midnight?

**Comment by sivonne** [comment]
I think it's due by tomorrow's class time?

**Comment by steven-king** [comment]
That is correct! Put it on the Oculus machine in the corner of the VR lab before class.

Do not be late for class because you are copying files.

On Mar 21, 2018, at 12:40 AM, sivonne <notifications@github.com<mailto:notifications@github.com>> wrote:


I think it's due by tomorrow's class time?

**Comment by sivonne** [comment]
Quick question, do we need to print our certificates for the drone training or are we submitting them online?

**Comment by yujie-tao** [comment]
Does it matter to export it for PC? or it should be Mac

**Comment by steven-king** [comment]
You are exporting for PC because Oculus only runs on PC.

Sent from my iPhone

On Mar 21, 2018, at 8:33 AM, Yujie Tao <notifications@github.com<mailto:notifications@github.com>> wrote:


Does it matter to export it for PC? or it should be Mac

**Comment by yujie-tao** [comment]
I only got a data folder after I exported it. Is that correct?

**Comment by cpa18** [comment]
Is there another solution to Inno Setup? It won't open for my Mac. Or is this step necessary for Oculus?

**Comment by steven-king** [comment]
You should be able to export from a Mac to PC. If that is not working you can copy your project to the PC in the lab and export from there.

To All… The expectation is to FIND A WAY. Be a problem solver. This new stuff is not easy. If it was, everyone would be doing it. Work through the problem. Find resources that explain how to fix it. The answers are out there.


On Mar 21, 2018, at 9:20 AM, cpa18 <notifications@github.com<mailto:notifications@github.com>> wrote:


Is there another solution to Inno Setup? It won't open for my Mac. Or is this step necessary for Oculus?

**Comment by steven-king** [comment]
There is a usability study going on in the VR lab today I was not aware of. So that we do not interrupt the study, please bring your exported files to class and we will copy them to an external hard drive. 

SO, you do NOT need to go put your files on the VR lab computer. If you have already, great job! 

Sorry for the change. See you in class.

**Comment by yujie-tao** [comment]
I think I left my USB flash drive over the computer in the corner. Can someone check that for me? It’s a small Sandisk with a oldwell keychain. I can pick it up later, thanks a lot!

**Comment by alexisbarnes** [comment]
@yujie-tao I have your USB. It will be in my office, suite 46-49.

**Comment by yujie-tao** [comment]
Can  i pick it up now?

Sent from my iPhone

On 21 Mar 2018, at 5:10 PM, Alexis Barnes <notifications@github.com<mailto:notifications@github.com>> wrote:


@yujie-tao<https://github.com/yujie-tao> I have your USB. It will be in my office, suite 46-49.

**Comment by alexisbarnes** [comment]
@yujie-tao Yes, I'm still in Reese.
